1 HW_Standard/HW_Hitmonchan.yaml

```diff
@@ -0,0 +1,21 @@
+species: Hitmonchan
+ingamename: Able
+displayname: SCP-076
+setname: Halloween
+tags: [halloween]
+item: Destiny Knot
+ability: Vital Spirit
+moves:
+    - [Outrage]
+    - [Rage]
+    - [Howl]
+    - [Uproar]
```

Comment:
Shouldn't howl be D as it's status?

Reply:
Didn’t know that there is a general moveset order. I’ll look into it, as well as the other changes. Thanks for letting me know, since I’m not the most knowledgable competitive-wise.

Reply:
There's a guide to move order in discussions, in the big guide.